Subject: liboil: don't check for assembler options and add compiler options
From: Michael Olbrich <m.olbrich@pengutronix.de>

Checking for "-Wa,-mfpu=vfp" and then adding "-mfpu=vfp" os of course nonsense.


Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>

---
 configure.ac |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -169,10 +169,10 @@ if test x$HAVE_GCC_ASM = xyes -a x$HAVE_
 fi
 
 if test x$HAVE_GCC_ASM = xyes -a x$HAVE_ARM = xyes ; then
-  AS_COMPILER_FLAG(["-Wa,-mfpu=vfp"],
+  AS_COMPILER_FLAG(["-mfpu=vfp"],
     [LIBOIL_CFLAGS="$LIBOIL_CFLAGS -mfpu=vfp"],
     true)
-  AS_COMPILER_FLAG(["-Wa,-mfloat-abi=softfp"],
+  AS_COMPILER_FLAG(["-mfloat-abi=softfp"],
     [LIBOIL_CFLAGS="$LIBOIL_CFLAGS -mfloat-abi=softfp"],
     true)
 fi
